pkg/application/service/services.go

```diff
@@ -19,7 +20,12 @@ type VerificationService interface {
 	VerifyCode(ctx *gin.Context, userID string, code string) error
 }
 
+type ToolchainClusterService interface {
+	GetNamespace(ctx *gin.Context, userID string) (*namespace.Namespace, error)
+}
+
 type Services interface {
 	SignupService() SignupService
 	VerificationService() VerificationService
+	ToolchainClusterService() ToolchainClusterService
```

Comment:
```suggestion
	MemberClusterService() MemberClusterService
```
since this service will look-up tokens on the member clusters

Reply:
Done in https://github.com/codeready-toolchain/registration-service/pull/224/commits/c44b99a8fc4b8ad059ed484cf3cf58cad94cb06d